recipes/Cbc/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "coin-or-cbc" %}
+{% set version = "2.10.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/coin-or/Cbc/archive/releases/{{ version }}.tar.gz
+    sha256: 601a7084e75955f98ccf1ef982d272cbe7f34558c527dadc661d49e6f0fc58a3
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make  # [unix]
+  host:
+    - coin-or-utils
+    - coin-or-cgl
+    - coin-or-clp
+    - coin-or-osi
+    - libblas
+    - libcblas
+    - liblapack
+    - readline  # [unix]
+    - zlib
+    - bzip2
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libCbc${SHLIB_EXT}
+    - test -f $PREFIX/include/coin/CbcConfig.h
+    - test -f $PREFIX/lib/libCbcSolver${SHLIB_EXT}
+    - test -f $PREFIX/lib/libOsiCbc${SHLIB_EXT}
+
+about:
+  home: https://projects.coin-or.org/Cbc
```

Comment:
The `home` URL should be the same as `dev_url` for all projects. We've migrated away from  https://projects.coin-or.org/ and are fully on Github now (that URL will eventually re-direct to Github). There is a user's guide of sorts at https://coin-or.github.io/Cbc/, but I guess I wouldn't point directly to that as the home page.